stdlib/out_channel.mli

```diff
@@ -158,3 +158,9 @@ val set_buffered : t -> bool -> unit
 val is_buffered : t -> bool
 (** [is_buffered oc] returns whether the channel [oc] is buffered (see
     {!set_buffered}). *)
+
+val isatty : t -> bool
+(** Returns [true] if the given channel refers to a terminal or console
```

Comment:
You could have retained the documentation style of `is_buffered` (i.e. `isatty oc` is `true` if the channel `oc` refers…). But I'm nitpicking.

Reply:
Wish granted, thanks.